data/redisrepo/order_iter.go

```diff
@@ -0,0 +1,82 @@
+package redisrepo
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+	"github.com/orbs-network/order-book/models"
+	"github.com/orbs-network/order-book/utils/logger"
+	"github.com/orbs-network/order-book/utils/logger/logctx"
+)
+
+// ////////////////////////////////////////////////
+type OrderIter struct {
+	index int
+	ids   []string
+	redis *redisRepository
+}
+
+func (i *OrderIter) Next() *models.Order {
+	ctx := context.Background()
```

Comment:
We can leave this for now, but this context should be from a function argument (eg. `Next(ctx context.Context)` so that we're using the same context all the way through the app

Reply:
agreed